### DISCUSSION TITLE
Can Daily Trading Mode work with EURO ?

### DISCUSSION THREAD

**Discussion by BlueCute** [Drakkar-Software/OctoBot#2042] [General]
Hello,

I want to ask if it is possible for Daily Trading Mode to work with EURO?
I have an account in Binance and ftx, and in both accounts my balance is in EURO.
Is this the reason I don't have a single transaction?
I use the following settings:
In simulation, it makes transfers, but when I launch real trading, nothing happens. I left it running all day and all night and it didn't do a single transaction.
In fact, how much is a good amount to start with?

Currencies:
Bitcoin: BTC/USDT
Ethereum: ETH/USDT
Solana: SOL/USDT
XRP: XRP/USDT

Exchanges:
binance
ftx

I will be very grateful if someone answers.

**Comment by BlueCute** [comment]
Finally !
Daily Trading started working :)
It took me a long time to read and learn.
And lots of experiments :)
It remains to learn the arbitrage settings.
It will be a bit difficult for me, no one helps with advice.

Octobot is wonderful, I am grateful to the creators.

**Reply by BlueCute** [reply]
There is only one thing I haven't been able to figure out yet about daily trading. After purchase, after how long does he sell (if the price has risen) or use cancel ? How much time it need ?

**Reply by GuillaumeDSM** [reply]
It's selling when it receives a sell signal. This signal comes from actuavated evaluators. For the RSIMomentumEvaluator for example, it's when fast and slow moving averages are crossing, the strengh of the signal depending on the value of the RSI.
To usually it is triggered by price yes.

**Comment by GuillaumeDSM** [comment]
> Hello,
> 
> I want to ask if it is possible for Daily Trading Mode to work with EURO? I have an account in Binance and ftx, and in both accounts my balance is in EURO. Is this the reason I don't have a single transaction? I use the following settings: In simulation, it makes transfers, but when I launch real trading, nothing happens. I left it running all day and all night and it didn't do a single transaction. In fact, how much is a good amount to start with?
> 
> Currencies: Bitcoin: BTC/USDT Ethereum: ETH/USDT Solana: SOL/USDT XRP: XRP/USDT
> 
> Exchanges: binance ftx
> 
> I will be very grateful if someone answers.

I guess you figured it out already but you need to add /EUR pairs to trade EUR (like BTC/EUR)
